src/repository/MongoDb.js

```diff
@@ -45,11 +45,18 @@ class MongoDb extends BaseMongoDb implements IBaseDatabase {
     await this.__runForCollection(
       collectionName,
       async (collection: Object): Promise<*> => {
-        const { page, pageSize = 25, ...otherQuery } = query;
+        const { page, pageSize = 25, skip, take, ...otherQuery } = query;
```

Comment:
Hmm.. we already have `page` and `pageSize`. Are these just for older APIs? You can probably give `take` a default value